packages/wallet-sdk/src/core/type/index.ts

```diff
@@ -28,8 +28,3 @@ export type RegExpString = OpaqueType<'RegExpString', string>;
 export const RegExpString = OpaqueType<RegExpString>();
 
 export type Callback<T> = (err: Error | null, result: T | null) => void;
-
-export type Chain = {
```

Comment:
we don't need this made-up type anymore. SCWSigner is the only one using this anyway.